plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -183,9 +197,23 @@ private static void generateEnumAttributeSchema(JSONObject attributeSchemaTempla
             }
             attributeSchemaTemplate.put(attribute.getName(),
                 new JSONObject()
+                    .put("description", retrieveDocStringFromAttribute(baseConfigurator.getTarget(), attribute.name))
                     .put("type", "string")
                     .put("enum", new JSONArray(attributeList)));
         }
     }
+
+    public static String retrieveDocStringFromAttribute(Class baseConfigClass, String attributeName) {
+        String htmlDocString = null;
+        try {
+            htmlDocString = ConfigurationAsCode.get().getHtmlHelp(baseConfigClass, attributeName);
```

Comment:
this method returns empty string when it can't find help but you're returning null on exception, might be best to always return null on empty, or even `Optional<String>` and then only include description where present?